### PR TITLE
Use cuda9 1 in c api packaging

### PR DESCRIPTION
**Pull request by shahasad**

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml [hunk 1]
```diff
@@ -152,4 +186,11 @@ jobs:
         buildConfig: $(buildConfig)
         artifactName: 'onnxruntime-win-gpu-$(buildArch)'
 
+    - task: PowerShell@2
+      displayName: 'Clean up Cuda Path 9.1'
+      inputs:
+        targetType: 'filePath'
+        filePath: '$(Build.SourcesDirectory)/tools/ci_build/github/windows/clean_up_cuda_prop_files.ps1'
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml]: Where would you run this script?
Can we avoid to do that? 

I manually installed the extensions to our github CI build machines, but it often get deleted by scripts like this.

File: tools/ci_build/build.py [hunk 2]
```diff
@@ -410,7 +410,7 @@ def setup_cuda_vars(args):
 
             cuda_bin_path = os.path.join(cuda_home, 'bin')
             os.environ["CUDA_BIN_PATH"] = cuda_bin_path
-            os.environ["PATH"] += os.pathsep + cuda_bin_path + os.pathsep + os.path.join(cudnn_home, 'bin')
+            os.environ["PATH"] = cuda_bin_path + os.pathsep + os.path.join(cudnn_home, 'bin') + os.pathsep + os.environ["PATH"]
```
Comment (snnn) [inline tools/ci_build/build.py]: Do not put anything before the %PATH%. It may create unexpected behavior.

Comment (snnn) [reply tools/ci_build/build.py]: Also, maybe this line is not needed.

**Comment by snnn** [comment]
Please use '--cuda_version' in build.py to choose between multiple installed CUDA installations.  I'm sure it works. I tested. And this is only needed for Windows. 
On Linux, we have docker, in the docker image, it should only have one cuda.

**Comment by shahasad** [comment]
> Please use '--cuda_version' in build.py to choose between multiple installed CUDA installations.  I'm sure it works. I tested. And this is only needed for Windows.
> On Linux, we have docker, in the docker image, it should only have one cuda.

did u mean I don't have to set up cuda_path and cudnn_path?
Either way the build is failing, and I have been trying different things... without success so far.

you may check the build log --
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=9606

File: tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml (hunk 1 above)
Comment (pranavsharma) [reply tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml]: let's revisit this after tomorrow's release.

File: tools/ci_build/build.py [hunk 3]
```diff
@@ -198,9 +195,8 @@ def install_ubuntu_deps(args):
         except Exception as e:
             raise BuildError("Error setting up required APT packages. {}".format(str(e)))
 
-def install_python_deps(numpy_version=""):
```
Comment (sdpython) [inline tools/ci_build/build.py]: Why was it removed?

File: tools/ci_build/build.py [hunk 4]
```diff
@@ -79,9 +79,6 @@ def parse_arguments():
     # Python bindings
     parser.add_argument("--enable_pybind", action='store_true', help="Enable Python Bindings.")
     parser.add_argument("--build_wheel", action='store_true', help="Build Python Wheel. ")
-    parser.add_argument("--numpy_version", default='1.15.0', help="Installs a specific version of numpy "
```
Comment (xadupre) [inline tools/ci_build/build.py]: Why was it removed?

**Pull request merged**